vue-app/src/components/Cart.vue

```diff
@@ -569,63 +570,52 @@ export default class Cart extends Vue {
   get errorMessage(): string | null {
     const currentUser = this.$store.state.currentUser
     const currentRound = this.$store.state.currentRound
-    if (!currentUser) {
-      return 'Please connect your wallet'
-    } else if (!this.isCorrectNetwork()) {
+    if (currentRound.messages >= currentRound.maxMessages)
```

Comment:
```suggestion
    if (this.$store.getters.isMessageLimitReached)
```
Might as well stick to the getters here